invoices/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Invoice(models.Model):
+    """ Model for invoices
+    """
+    DUE_RECEIPT = 'Due of Receipt'
```

Comment:
i said fix the value of this field.

`due_receipt`, `due_end_next_month`, `due_end_month`